Tools/TestWebKitAPI/Tests/WebKitCocoa/MediaSession.mm

```diff
@@ -98,7 +99,7 @@ void TearDown() override
 
     TestWKWebView* webView() { return _webView.get(); }
 
-    pid_t webViewPid() { return [_webView _webProcessIdentifier]; }
+    pid_t webViewPid() { return [_webView _gpuProcessIdentifier]; }
```

Comment:
Is this intentional?

In any case, although it isn’t new to your patch this function name seems wrong

Reply:
It is intentional, because we check that the Now Playing app's PID equals `webViewPid()` (which it no longer does now that we use MediaRemote via GPUP). Good call though, I'll fix the name.